README.md

```diff
@@ -593,6 +615,19 @@ that refers to them.
 ### Why are some typedefs not generated?
 
 The following typedefs are not generated -
-- They are not referred to anywhere in the included declarations.
-- They refer to a struct/union having the same name as itself.
-- They refer to a boolean, enum, inline array, Handle or any unsupported type.
+  - They are not referred to anywhere in the included declarations.
+  - They refer to a struct/union having the same name as itself.
+  - They refer to a boolean, enum, inline array, Handle or any unsupported type.
+
+### What are these logs generated by ffigen and how to fix them?
+
+Ffigen can sometimes generate a lot of logs, especially when it's parsing a lot of code.
+- `SEVERE` logs are something you *definitely need to address*. They can be caused due
+  to syntax errors, or more generally missing header files (which need to be specified
+  using `compiler-opts` in config)
+- `WARNING` logs are something *you can ignore*, but should probably look into. These are mostly
+  indications of declarations ffigen couldn't generate due to limitations of dart:ffi,
+  private declarations (which can be resolved by renaming them via ffigen config) or other minor issues
+  in the config file itself.
+- Everything else can be safely ignored. It's purpose is to simply
+  let you know what ffigen is doing.
```

Comment:
Maybe add a `-` with how to change the verbosity.

Reply:
Done